synfig-studio/src/gui/render.cpp

```diff
@@ -451,7 +452,7 @@ RenderSettings::check_target_destination()
 					n_frame++)
 			{
 				if(Glib::file_test(filename_sans_extension(filename) +
-					tparam.sequence_separator + 
```

Comment:
It looks like this change is redundant because you have already updated the `tparam.sequence_separator`.

Reply:
That's right, not necessary :+1: